wdl-lint/tests/lints/missing-preamble-ws/source.wdl

```diff
@@ -1,6 +1,5 @@
 #@ except: DescriptionMissing
 ## This is a test of missing preamble whitespace.
-
```

Comment:
This looks like an unintended addition.

Reply:
This reverts a mistaken change in a prior PR. There should be no whitespace there, so that the preamble whitespace rule fires.

Reply:
Here: https://github.com/stjude-rust-labs/wdl/commit/1e289b967673bad25504e33f35733d907a2d6467#diff-df8d360942f292702a6c79b38bf6e6af9db95c7dd8d4879eb58ae349b8110b87

Reply:
Oh okay, sounds good.